crates/core_arch/src/x86/sse.rs

```diff
@@ -194,7 +194,7 @@ pub unsafe fn _mm_min_ss(a: __m128, b: __m128) -> __m128 {
 #[cfg_attr(test, assert_instr(minps))]
 #[stable(feature = "simd_x86", since = "1.27.0")]
 pub unsafe fn _mm_min_ps(a: __m128, b: __m128) -> __m128 {
-    minps(a, b)
+    simd_fmin(a, b)
```

Comment:
`simd_fmin` get lowered to [`minnum`](https://llvm.org/docs/LangRef.html#llvm-minnum-intrinsic).The only possible difference with `minps` it the behaviour with `-0.0` and `0.0`. LLVM doesnt guarantee that one is smaller than the other, while `minps` always picks the second operand it seems. The intrinsic docs at intel unhelpfully list the result as `MIN(a, b)`

Reply:
Maybe we can add a test for this and verify that it passes before this change and after it ?

cc @rkruppe

Reply:
I reverted the change, as `simd_fmin` does behave differently from `minps` for `-0.0`.

Reply:
I'm resolving this.

 If you want to improve this, you might need to add a different `simd_` intrinsic to rust, that instead of generating a `llvm.minnum`, it instead generates an `fcmp ult` followed by a `select`. I recommend checking out first whether it is at all possible to match `minps` semantics with that, e.g., depending on how these handle denormals, and the case where one of the arguments is a nan.